.cirrus.yml

```diff
@@ -0,0 +1,35 @@
+connected_check_task:
+  name: Run Android instrumented tests
+  env:
+    API_LEVEL: 28
```

Comment:
ça me parait vachement haut ça non ? Est-ce que c'est possible de build sur plusieurs api levels en parallèle ?

Reply:
Je peux réduire l'API level mais à combien? Ca serais possible mais ça demanderais de run plusieurs émulateurs en même temps. Donc même avec 8 cores je pense pas qu'on puisse run plein d'emu en même temps.

Reply:
https://dev.to/ychescale9/exploring-cirrus-ci-for-android-434

Reply:
Comme CirrusCI utilise macOS pour l'hardware acceleration on est limité à une vm. https://cirrus-ci.org/faq/#are-there-any-limits